.github/workflows/contributor-pr.yml

```diff
@@ -56,7 +47,7 @@ jobs:
               compileAll
               --no-configuration-cache
               -DdisableLocalCache=true
-              ${{ steps.determine-sys-prop-args.outputs.sys-prop-args }}
+              -DcacheNode=us
```

Comment:
🤔 Does this mean we wouldn't publish a scan for contributor builds any more since they don't have the publishing credenentials?